client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/classes/FloatClassReplacement.java

```diff
@@ -0,0 +1,96 @@
+package org.evomaster.client.java.instrumentation.coverage.methodreplacement.classes;
+
+
+import org.evomaster.client.java.instrumentation.coverage.methodreplacement.MethodReplacementClass;
+import org.evomaster.client.java.instrumentation.coverage.methodreplacement.Replacement;
+import org.evomaster.client.java.instrumentation.heuristic.Truthness;
+import org.evomaster.client.java.instrumentation.shared.ReplacementType;
+import org.evomaster.client.java.instrumentation.shared.StringSpecialization;
+import org.evomaster.client.java.instrumentation.shared.StringSpecializationInfo;
+import org.evomaster.client.java.instrumentation.shared.TaintInputName;
+import org.evomaster.client.java.instrumentation.staticstate.ExecutionTracer;
+
+import static org.evomaster.client.java.instrumentation.coverage.methodreplacement.DistanceHelper.*;
+
+public class FloatClassReplacement implements MethodReplacementClass {
+
+    @Override
+    public Class<?> getTargetClass() {
+        return Float.class;
+    }
+
+
+    @Replacement(type = ReplacementType.EXCEPTION, replacingStatic = true)
+    public static float parseFloat(String input, String idTemplate) {
+
+        if (TaintInputName.isTaintInput(input)) {
+            ExecutionTracer.addStringSpecialization(input,
+                    new StringSpecializationInfo(StringSpecialization.INTEGER, null));
```

Comment:
add TODO for `StringSpecialization.DOUBLE`

Reply:
ok